client/packages/system/src/Name/ListView/Facilities/ListView.tsx

```diff
@@ -9,11 +9,15 @@ import {
   DotCell,
   ColumnAlign,
   useEditModal,
+  useToggle,
 } from '@openmsupply-client/common';
-import { useName, NameRowFragment } from '../../api';
+import { useName } from '../../api';
 import { NameRenderer } from '../../Components';
 import { Toolbar } from './Toolbar';
 import { FacilityEditModal } from './FacilityEditModal';
+import { AppBarButtons } from './AppBarButtons';
+import { PropertiesImportModal } from '../ImportProperties/PropertiesImportModal';
+import { FacilityNameRowFragment } from '../../api/operations.generated';
```

Comment:
Have added a new nameRowFragment which includes properties along with a new query for facilities to include properties.

I wasn't sure whether best to do this, or to add properties to all other nameRowFragment calls.